src/Microsoft.Security.Utilities.Core/RegexPattern.cs

```diff
@@ -403,4 +412,4 @@ private protected static string MakeHighPerformancePattern(string pattern, strin
         return "^" + pattern;
     }
 #endif
-}
+}
```

Comment:
You can leave this, but just noting that doing #153 would prevent final-newline inconsistency from happening.